src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com"
+      method="get"
+      class="form-big"
+      data-qa="keypress"
+    >
+      <input
+        type="search"
+        class="input-big"
```

Comment:
big input still has the wrong width in the demo:
![image](https://user-images.githubusercontent.com/86923089/216816030-9f530f35-072c-41ca-8761-adfe1f5a3278.png)

make sure you've done deploy of the latest changes